health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/AddOrEditMapping.js

```
@@ -1,196 +1,217 @@
+              setNewData(updatedData);
+            }}
+            placeholder={t("")}
+          />
+        </div>
+      );
+    } else if (column?.type === "string") {
+      if (column?.description === "Boundary Code" || column?.description === "Boundary Code (Mandatory)") {
+        return (
+          <div style={{ marginBottom: "1rem" }}>
+            <div style={{ display: "flex" }}>
+              <label>{t("CHOOSE_BOUNDARY_LEVEL")}</label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)